.gitignore

```diff
@@ -1,3 +1,5 @@
+.gitignore
+
 # reated by https://www.toptal.com/developers/gitignore/api/node
 # Edit at https://www.toptal.com/developers/gitignore?templates=node
```

Comment:
was is das?

Reply:
copy pasted your .gitignore file for node and those lines came with it